src/gt4py/backend/gtc_backend/gtcpp/backend.py

```diff
@@ -160,27 +169,12 @@ def apply(cls, root, *, module_name="stencil", **kwargs) -> str:
         return formatted_code
 
 
-@gt_backend.register
-class GTCGTBackend(BaseGTBackend, CLIBackendMixin):
-    """GridTools python backend using gtc."""
-
-    name = "gtc:gt:cpu_ifirst"
-
-    GT_BACKEND_T = "x86"
+class GTCGTBaseBackend(BaseGTBackend, CLIBackendMixin):
     options: ClassVar[Dict[str, Any]] = {}
```

Comment:
unrelated to your work, but fits well to this PR:
```suggestion
    options = BaseGTBackend.GT_BACKEND_OPTS
```

Reply:
test if it works by passing `verbose=True` in the decorator.